### PR TITLE
docs: Add release schedule for UnidirectionalTopicOperator feature gate

### LINKED ISSUE #8935: [Docs]: Release schedule for UnidirectionalTopicOperator feature gate
### Suggestion / Problem

We have started testing UTO in some of our dev/test environments by following the guidance [here](https://strimzi.io/docs/operators/latest/deploying.html#ref-operator-unidirectional-topic-operator-feature-gate-str). But I had some trouble finding the tentative release schedule of the feature gate. The table [here](https://strimzi.io/docs/operators/latest/deploying.html#ref-operator-unidirectional-topic-operator-feature-gate-str:~:text=longer%20be%20disabled.-,Table%2029,-.%20Feature%20gates%20and) doesn't seem to have an entry for the UTO feature gate. The proposal does seem to mention a [graduation timeline](https://github.com/strimzi/proposals/blob/main/051-unidirectional-topic-operator.md#co-managed-case-1) but I am not sure if it's up to date as the feature gate was suppose to be in alpha phase in 0.35 according to it. 
It would be great if the docs could be updated with the release schedule of the UTO feature gate following the recent developments. It will help us to prepare ourselves better for any future changes in regards to Topic operator.      

### Documentation Link

https://strimzi.io/docs/operators/latest/deploying.html#ref-operator-unidirectional-topic-operator-feature-gate-str:~:text=longer%20be%20disabled.-,Table%2029,-.%20Feature%20gates%20and

### PR DESCRIPTION
**Pull request by Kartikey-Mishra1**
### Type of change
- Documentation

### Description

Add UnidirectionalTopicOperator feature gate release schedule in `Feature gate releases` section and the feature gates and Strimzi versions table

Closes #8935

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [x] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

**Review by ppatierno — APPROVED** [review]
LGTM. Thanks!

File: documentation/modules/operators/ref-operator-cluster-feature-gate-releases.adoc [hunk 1]
```diff
@@ -67,6 +69,11 @@ NOTE: Feature gates might be removed when they reach GA. This means that the fea
 ¦0.39 (planned)
 ¦ -
 
+¦`UnidirectionalTopicOperator`
+¦0.36
+¦0.38 (planned)
```
Comment (scholzj) [inline documentation/modules/operators/ref-operator-cluster-feature-gate-releases.adoc:74]:
```suggestion
¦0.39 (planned)
```

File: documentation/modules/operators/ref-operator-cluster-feature-gate-releases.adoc [hunk 2]
```diff
@@ -25,6 +25,8 @@ Alpha and beta stage features are removed if they do not prove to be useful.
   It is expected to move to GA phase and be always enabled from Strimzi 0.39.
 * The `KafkaNodePools` feature gate is in alpha stage and is disabled by default.
   It is expected to move to beta phase and be enabled by default from Strimzi 0.39.
+* The `UnidirectionalTopicOperator` feature gate is in alpha stage and is disabled by default.
+  It is expected to move to beta phase and be enabled by default from Strimzi 0.38.
```
Comment (scholzj) [inline documentation/modules/operators/ref-operator-cluster-feature-gate-releases.adoc:29]:
```suggestion
  It is expected to move to beta phase and be enabled by default from Strimzi 0.39.
```

**Review by scholzj — CHANGES_REQUESTED** [review]
We need to adjust this based on the delay in the introduction -> e.g. it shoud be still 3 releases in the alpha stage, just instead of 0.35, 0.36 and 0.37, it should be 0.36, 0.37 and 0.38. So it will move to beta in 0.39. Sorry if I somehow confused you in the discussion in the issue.

**Review by scholzj — APPROVED** [review]

**Comment by scholzj** [comment]
Thanks for the PR @Kartikey-Mishra1

**Pull request merged**